pkg/pillar/cmd/nim/nim.go

```diff
@@ -1054,10 +1054,13 @@ func (n *nim) makeLastResortDPC() (types.DevicePortConfig, error) {
 
 func (n *nim) includeLastResortPort(ifAttrs netmonitor.IfAttrs) bool {
 	ifName := ifAttrs.IfName
+	ifType := ifAttrs.IfType
 	exclude := strings.HasPrefix(ifName, "vif") ||
 		strings.HasPrefix(ifName, "nbu") ||
 		strings.HasPrefix(ifName, "nbo") ||
-		strings.HasPrefix(ifName, "keth")
+		strings.HasPrefix(ifName, "wlan") ||
+		strings.HasPrefix(ifName, "keth") ||
+		ifType == "can" || ifType == "vcan"
```

Comment:
I noticed that we already have some checks for `ifAttrs.IfType` below. Maybe they could be merged into one switch (together with can) and we leave this if statement for ifName comparisons?